doc/source/howtos/proxysql-v2.rst

```diff
@@ -0,0 +1,816 @@
+- The command ``--disable`` accepts the ``--writer-hg`` option to disable the Galera cluster associated with that hostgroup overriding the value specified in the configuration file.
+
+Removed Features
+--------------------------------------------------------------------------------
+
+- Asynchronous slave reader support has been removed: the ``--include-slaves`` option is not supported.
+- A list of nodes in the priority order is not supported in |proxysql| v2. Only a single node is supported at this time.
+- Since the galera_proxysql_checker and galera_node_monitor scripts are no longer run in the scheduler, automatic cluster membership updates are not supported.
+- Checking the pxc_maint_mode variable is no longer supported
+- Using desynced nodes if no other nodes are available is no longer supported.
+- The server status is no longer maintained in the mysql_servers table.
+
+Limitations
+--------------------------------------------------------------------------------
+
+- With ``--writers-as-readers=backup`` read-only nodes are not allowed. This a
+  limitation of ProxySQL 2.0.  Note that backup is the default value of
+  ``--writers-as-readers`` when ``--mode=singlewrite``
+
+Installing ProxySQL v2
+================================================================================
+
+If that is what you used to :ref:`install PXC <install>` or any other Percona
+software, run the corresponding command:
+
+* Installing on Debian or Ubuntu:
+
+.. code-block:: bash
+
+   $ sudo apt-get install proxysql2
+
+* Installing on Red Hat Enterprise Linux or CentOS:
+
+.. code-block:: bash
+
+   $ sudo yum install proxysql2
+
+Alternatively, you can download the packages from
+https://www.percona.com/downloads/proxysql/.
+
+To start ProxySQL, run the following command:
+
+.. code-block:: bash
+
+   $ sudo service proxysql2 start
+
+.. _default-credentials:
+
+.. warning::
+
+   **Do not run |proxysql| with default credentials in production.**
+
+   Before starting the ``proxysql`` service, you can change the
+   defaults in the :file:`/etc/proxysql.cnf` file by changing the
+   ``admin_credentials`` variable.  For more information, see `Global
+   Variables
+   <https://github.com/sysown/proxysql/blob/master/doc/global_variables.md>`_.
+
+Automatic Configuration
+================================================================================
+
+The ``proxysql2`` package from Percona includes the ``proxysql-admin`` tool for
+configuring |PXC| nodes with ProxySQL.
+
+Before using the ``proxysql-admin`` tool, ensure that ProxySQL and |PXC| nodes
+you want to add are running. For security purposes, please ensure to change the
+default user settings in the ProxySQL configuration file.
+
+.. important::
+
+   The ``proxysql-admin`` tool can only be used for *initial* ProxySQL
+   configuration.
+
+   The *ProxySQL Admin* (|proxysql-admin| tool) is specially developed by
+   Percona to automatize this configuration. Bug reports and feature proposals
```

Comment:
same as above, 'automatize this configuration' should be 'automate the configuration'